tests/api_workflow/test_api_workflow_compute_worker.py

```diff
@@ -61,9 +61,7 @@ def test_delete_compute_worker(self):
     def test_create_compute_worker_config(self):
         config_id = self.api_workflow_client.create_compute_worker_config(
             worker_config={
-                "stopping_condition": {
-                    "n_samples": 10,
-                },
+                "pretagging": True,
```

Comment:
Could you change this to set something of the new config?
E.g. 
```suggestion
                "training": {"task_name": "lightly_pretagging"},
```